Soccar [Game-Server]/src/nl/soccar/gameserver/Main.java

```diff
@@ -23,6 +25,11 @@
     private GameServerRmiController rmiController;
     private GameServerSocnetController socnetController;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)